test/api/controller/document.test.js

```diff
@@ -1030,6 +1030,78 @@ describe('DocumentController', () => {
     });
   });
 
+  describe('#deleteFields', () => {
+    let content;
```

Comment:
(Really nitpicking)
As far as I understand, this variable represents the request body so maybe the name `body` would be more appropriate